src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/IntegrationTests/HomeServiceTests.cs

```diff
@@ -22,22 +24,40 @@ public HomeServiceTests()
         [Fact]
         public void GetCorrectAnswerByUserId_UserAnsweredCorrectly_ReturnsUserTestCorrectAnswerDto()
         {
+            /*
             var testCorrectAnswer = GetTestAnswer();
             var test = GetTest();
+            */
+            var fixture = new Fixture();
+            fixture.Behaviors.OfType<ThrowingRecursionBehavior>().ToList()
+                .ForEach(b => fixture.Behaviors.Remove(b));
+            fixture.Behaviors.Add(new OmitOnRecursionBehavior());
+
+            fixture.Customize<UserTestCorrectAnswer>(c =>
+                c.With(ut => ut.TestId, 1)
+                    .With(ut => ut.UserId, TestUserId));
+
+            fixture.Customize<UserTestWrongAnswer>(c =>
+                c.With(ut => ut.TestId, 1)
+                    .With(ut => ut.UserId, TestUserId));
+
+            fixture.Customize<Test>(c =>
+                c.With(t => t.Id, 1));
+
+            var test = fixture.Create<Test>();
```

Comment:
There is a problem with the "test" variable changing. When you debug the test, after creating a test variable, its value corresponds to customization, but when you step inside using statement it changes.
![image](https://user-images.githubusercontent.com/28907927/91197433-98070c80-e6fb-11ea-9f10-d1fca7cd6c0c.png)

Reply:
I don't see, where you set ex. Start/EndDate

Reply:
maybe it has different Id? How many tests have you in db?

Reply:
The database is empty at the begging of every test. The problem is that EF method `context.Tests.Add(test);` is modifying "test" variable and puts in it default autofixture values, without our customizations.

Reply:
But even if that will work, we're inserting in database user three correct answers for only one test, which is incorrect behavior.

Reply:
"AutoFixture makes it easier for developers to do Test-Driven Development by automating non-relevant Test Fixture Setup, allowing the Test Developer to focus on the essentials of each test case."
And I suggest sticking to that because in those integration tests those arrange phases maybe aren't the most important, but with the current design I think are more readable, maintainable and are closer to real app behavior.